### ISSUE TITLE
@sophrom maybe you can add option where it runs script for wlan0,wlan1,wlan2 to boost TX power some thing like 

### ISSUE THREAD

**Issue by JustMev** [sophron/wifiphisher#418]
Sophron you could also implement TX script to increase power  some like this I run prior

`echo " taking down wlan0"
ifconfig wlan0 down
sleep 3

echo "setting Region to Guyana"
iw reg set GY
sleep 3

#iwconfig wlan0 channel 13
echo "setting TxPower to 30"
iwconfig wlan0 txpower 30
sleep 2

echo "starting wlan0"
ifconfig wlan0 up 
echo "pulling wlan0 interface up"
iwconfig
sleep 5

echo "       *********************************    "
echo "       *               BY              *    "
echo "       *                               *    "
echo "       *          JustMe ROMs          *    "
echo "       *  www.facebook.com/justmeroms  *    "
echo "       *              AeK              *    "
echo "       *                               *    "
echo "       *********************************    "

sleep 1

echo "Enjoy!"`

**Comment by sophron** [comment]
Yes, PyRIC has methods that we should leverage to take advantage of our network's card capacity.

**Comment by blackHatMonkey** [comment]
It is true that PyRIC has the methods to change TX power however as the author himself noted it does **not** work on all devices. The solutions seems to [this](http://askubuntu.com/questions/597546/iwconfig-wlan0-txpower-30mw-not-working) which I believe is out of scopes of Wifiphisher.
